### PR TITLE
Compatibility with Rails 5.1

### PR DESCRIPTION
**Pull request by pschambacher**
/cc @grosser

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,19 +4,18 @@ bundler_args: ""
 branches:
   only: master
 rvm:
-  - 2.0.0
-  - 2.1.8
-  - 2.2.4
+  - 2.2.7
```
Comment (grosser) [inline .travis.yml]: bump ruby version dependency ?

**Comment by pschambacher** [comment]
💇‍♂️  you're owner of this repo so I let you merge/bump/release

**Comment by grosser** [comment]
guess who just got added as collaborator :trollface:

File: lib/cia/auditable.rb [hunk 2]
```diff
@@ -6,7 +6,15 @@ def self.included(base)
...
-      changes
+      if respond_to?(:has_changes_to_save?)
+        if has_changes_to_save?
+          changes_to_save
+        else
+          saved_changes
+        end
+      else
+        changes
+      end
```
Comment (vanchi-zendesk) [inline lib/cia/auditable.rb]: http://ashleyangell.com/2019/05/changes-to-activerecorddirty-in-rails-5-2/

for future reference.

**Pull request merged**